templates/field/field--node--oe-news-contacts--oe-news--full.html.twig

```diff
@@ -0,0 +1,22 @@
+{#
+/**
+ * @file
+ * News contacts field template.
+ *
+ * @see ./core/themes/stable/templates/field/field.html.twig
+ */
+#}
+<div id="news-contacts">
+  <h2 class="ecl-u-type-heading-2">
+    {{ label }}
+  </h2>
+  {% if multiple %}
+  <div>
```

Comment:
Do we need empty div?
There is templates/field/field--node--oe-project-contact--oe-project--full.html.twig with output of Contact entity. Please have a look.

Reply:
Updated to resemble templates/field/field--node--oe-project-contact--oe-project--full.html.twig